README.md

```diff
@@ -605,6 +612,38 @@ This returns one row for every `Animal` whose list of aliases contains the value
 for the `$wanted` parameter. Each row contains the matching `Animal`'s name
 in a column named `animal_name`.
 
+#### Constraints and Rules
+- Must be on a property field of list type.
+
+### has_edge_degree
+#### Example Use
+```
+{
+    Animal {
+        name @output(out_name: "animal_name")
+
```

Comment:
Is there a reason for the newline here? I don't see it anywhere else in the docs under a similar context.

Reply:
It's not necessary, it just visually separates the output from the meat of the query with the complicated filter. Would you prefer it removed?

Reply:
I mostly care about consistency throughout the docs. It is a pretty complicated filter so maybe it's good to keep it.